ui/src/components/debug_logger/mod.rs

```diff
@@ -34,49 +64,165 @@ pub fn DebugLogger(cx: Scope) -> Element {
+                                appearance: if filter_level.get() == &LogLevel::Error { Appearance::Danger } else { Appearance::Secondary },
+                                onpress: |_| {
+                                    filter_level.set(LogLevel::Error);
                                 },
-                                span {
-                                    class: "log-text muted",
-                                    "»"
-                                }
-                                span {
-                                    aria_label: "debug-logger-item-text",
-                                    id: "log_text",
-                                    class: "log-text",
-                                    " {log_message}"
-                                }
+                                tooltip: cx.render(rsx!(
+                                    Tooltip {
+                                        arrow_position: ArrowPosition::Top,
+                                        text: "Error".into()
+                                    }
+                                )),
+                            },
+                            Button {
+                                icon: Icon::BugAnt,
+                                appearance: if filter_level.get() == &LogLevel::Debug { Appearance::Secondary } else { Appearance::Secondary },
+                                onpress: |_| {
+                                    filter_level.set(LogLevel::Debug);
+                                },
+                                tooltip: cx.render(rsx!(
+                                    Tooltip {
+                                        arrow_position: ArrowPosition::Top,
+                                        text: "Debug".into()
+                                    }
+                                )),
+                            },
+                        }
+                    })),
+                    Button {
+                        text: "State".into(),
+                        icon: Icon::Square3Stack3d,
+                        appearance: if active_tab.get() == "State" { Appearance::Primary } else { Appearance::Secondary },
+                        onpress: |_| {
+                            active_tab.set("State".into());
+                        }
+                    },
+                    Button {
+                        text: "Web Inspector".into(),
+                        icon: Icon::ArrowTopRightOnSquare,
+                        appearance: Appearance::Secondary,
+                        onpress: |_| {
+                            window.webview.open_devtools();
+                        }
+                    },
+                },
+                div {
+                    class: "logger-nav-right",
+                    aria_label: "debug-logger-nav-right",
+                    IconElement {
+                        icon: if state.read().ui.theme.clone().unwrap_or_default().name == "Light" {
```

Comment:
fyi this would be more elegant: `ui.theme.map(|theme| theme.name == "Light").unwrap_or_default()`